index.js

```diff
@@ -46,8 +51,9 @@ async function callCumulusMessageAdapter(command, input) {
     throw new CumulusMessageAdapterExecutionError(stderr);
   }
   catch (error) {
-    const msg = `CMA process failed to run, check that python runtime is present in the path and/or
-    the CMA package is present in ${adapterDir}.   Error: ${error}`;
+    const msg = `CMA process failed (${error.shortMessage})\n
```

Comment:
execa includes stderr and/or a full message in it's errors (optionally a shortMesaage as of 4.0).   In the case of CMA failures, more is probably better.